### ISSUE TITLE
10-7959a: sponsor information

### ISSUE THREAD

**Issue by marywang2** [department-of-veterans-affairs/va.gov-team#86839]
As a VA.gov user, I want to fill out the CHAMPVA claim form with the sponsor information so that I can file a claim.

[Figma link](https://www.figma.com/design/Tfhq5h2LwXEeEEtFBAAFOv/CHAMPVA-Claims-(10-7959a)?node-id=961-139667&t=Ko13IszQmJiM9ZRY-0)

- Page elements:
  - breadcrumb 
  - header and subhead
  - copy
  - footer
- Sponsor information
  - name/dob

Acceptance criteria:

- Given the user is on VA.gov and about to fill out the CHAMPVA claim form
- When the user gets to the sponsor info page
- Then the user can fill out the sponsor's information

**Comment by michaelclement** [comment]
Work is done [in this PR](https://github.com/department-of-veterans-affairs/vets-website/pull/30566). Waiting on:

- [ ] Team approval
- [ ] Platform approval

**Comment by michaelclement** [comment]
<img width="532" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNXVWQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--5f6e194b6593d3ab43d1d3dec8dbe843d733b78f/Screenshot%202024-06-26%20at%2017.21.46.png" alt="Screenshot 2024-06-26 at 17.21.46.png" />


<img width="758" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNXFWQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--ed1d6d3f5a4841002a8cc2a0f5d271992a0d59ae/Screenshot%202024-06-26%20at%2017.26.40.png" alt="Screenshot 2024-06-26 at 17.26.40.png" />

**Comment by michaelclement** [comment]
@marywang2 This has been merged and should be on staging shortly. Thanks!

**Comment by marywang2** [comment]
@michaelclement do you have the staging url for this one? Really small thing but would you please add a period at the end of the sentence (after "eligibility")? (@syd-hoeper for review as well).

**Comment by michaelclement** [comment]
@marywang2 @syd-hoeper staging url: https://staging.va.gov/ivc-champva/10-7959a - 

@marywang2 Yep! I'll add that period in another PR. Will that block closing this one, do you think?

**Comment by marywang2** [comment]
Thanks! I can close this one now but @syd-hoeper fyi - please chime in if you have any other changes.

**Comment by michaelclement** [comment]
Thanks!
